src/core/ProgressBar/__tests__/ProgressBar.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import { ProgressBar } from '../ProgressBar';
+import { screen } from '@testing-library/dom';
+
+afterEach(cleanup);
+jest.useFakeTimers();
+
+describe('ProgressBar renders', () => {
+  it('Render', () => {
+    // const items = [{ value: 80, color: 'rgb(63, 81, 181)', label: 'Success' }];
+    const { getByText } = render(
+      <ProgressBar value={80} color={'red'} label="Success" />
+    );
+    const elem = screen.getByRole('progressbar');
+    // check value progressbar
+    expect(elem.getAttribute('aria-valuenow')).toBe('80');
+    // check color progressbar
+    const el = document.getElementsByClassName('MuiLinearProgress-bar');
+    const style = window.getComputedStyle(el[0]);
+    expect(style.backgroundColor).toBe('rgb(63, 81, 181)');
```

Comment:
Can we use the actual theme colors of red, green, generic type progress bar in the tests instead of primary red rgb!

Reply:
This comment is unaddressed

Reply:
@S-ayanide I need to try

Reply:
Instead of `color={'red'}` you can use `color={useTheme().palette.primary.main}` or something and at the bottom assert it with `toBe(<hex code of the theme property>)`

Reply:
Can we remove the progress bar background color checking test for now, because it has no meaning if we put 'red' and check its rbg color in test. It better to not have it now and later have the check from the theme colors instead.